sample-apps/java/client/devserver-client-sample/src/main/java/timefetch/TimeWorker.java

```diff
@@ -0,0 +1,33 @@
+package timefetch;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.worker.Worker;
+import io.temporal.worker.WorkerFactory;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+
+public class TimeWorker {
+    public static void runWorker(String[] args) {
+        // Create Developer Server local service stub
+        WorkflowServiceStubs service = WorkflowServiceStubs.newLocalServiceStubs();
+
+        // A Workflow Client can start, Signal, and Query a Workflow Execution
+        WorkflowClient client = WorkflowClient.newInstance(service);
+        
+        // A Workflow Factory creates workers
+        WorkerFactory factory = WorkerFactory.newInstance(client);
+        
+        // A Worker listens to one task queue, processing workflows and activities
+        Worker worker = factory.newWorker("TimeFetchQueue");
+
+        // Register a Workflow implementation with this worker
+        // The implementation must be known at runtime to dispatch workflow tasks
+        worker.registerWorkflowImplementationTypes(TimeWorkflowImpl.class);
+
+        // Register Activity Types from the Activity vendor with the worker
```

Comment:
The type vends Activities that can be dynamically invoked. I think that's fairly industry standard in production. It's moot here, because semantics are removed, but again I would love to talk standard terms for the team.

Thank you again for thoughtful, clear, and actionable feedback even if I cannot use it in this case as the semantics are removed from the sample.